mmeval/metrics/dota_map.py

```diff
@@ -0,0 +1,294 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import logging
+import numpy as np
+from multiprocessing.pool import Pool
+from typing import Dict, List, Optional, Sequence, Tuple
+
+from mmeval.metrics.voc_map import VOCMeanAP
+from .utils.bbox_overlaps_rotated import calculate_bboxes_area_rotated
+
+logger = logging.getLogger(__name__)
+try:
+    # we prefer to use bbox_iou_rotated in mmcv
+    from mmcv.ops import box_iou_rotated as calculate_overlaps_rotated
+except Exception as e:  # noqa F841
+    logger.info(
+        'mmcv is not installed, using iou calculating method with OpenCV.')
+    from .utils.bbox_overlaps_rotated import calculate_overlaps_rotated
```

Comment:
This should moved to `mmeval/metrics/utils/bbox_overlaps_rotated.py`, and log the warning only when `calculate_overlaps_rotated` is used.